src/python/grapl_provision/grapl_provision.py

```diff
@@ -405,12 +407,13 @@ def sqs_provision_loop() -> None:
                     local_dg_provision_client,
                 )
                 mg_succ = True
-                print("Provisioned mastergraph")
+                print("Provisioned master graph")
                 break
         except Exception as e:
             if i > 10:
                 LOGGER.error("mg provision failed with: ", e)
 
+    print("Starting to provision Secrets Manager")
```

Comment:
just to help the user debug in case provision isn't working

Reply:
Given that we have a `LOGGER` I'd prefer these prints were instead `LOGGER.info(..)`